launcher/ui/processlist.h

```diff
@@ -47,7 +47,7 @@ struct ProcData
     QString user;
     GammaRay::ProbeABI abi;
 
-    ProcData() {}
+    ProcData() = default;
```

Comment:
just remove this line altogether (rule of zero)

Reply:
done